### ISSUE TITLE
Uncaught TypeError: Cannot use 'in' operator to search for 'mc' in 1286

### ISSUE THREAD

**Issue by sandilya28** [ruilisi/fortune-sheet#137]
**Describe the bug**

I'm trying to show a SQL query output in a fortune-sheet worksheet. I'm rendering the worksheet using the following code

```
<Workbook
  showToolbar={false}
  showSheetTabs={false}
  data={[
    {
      name: "Sheet1",
      celldata: cellData,
    },
  ]}
/>

```
There is a glimpse of the worksheet being rendered for a second and the app crashes with the following exception,

```
Uncaught TypeError: Cannot use 'in' operator to search for 'mc' in 1286
    at mergeBorder (index.esm.js:48304:1)
    at getCellRowColumn (index.esm.js:55025:1)
    at index.esm.js:2247:1
    at index.esm.js:2260:1
    at e.produce (immerClass.ts:94:1)
    at e.produceWithPatches (immerClass.ts:141:1)
    at index.esm.js:7438:1
    at basicStateReducer (react-dom.development.js:16540:1)
    at updateReducer (react-dom.development.js:16664:1)
    at updateState (react-dom.development.js:17004:1)
```

**To Reproduce**
1. Render the sheet
2. Wait for a couple of seconds before the app is crashed

**Expected behavior**
The worksheet shouldn't crash. 

**Screenshots**
![ezgif com-gif-maker (5)](https://user-images.githubusercontent.com/6747531/208588733-03901fb7-c045-4472-b8a3-bbbd3ffedb23.gif)

**Comment by sandilya28** [comment]
Code for creating the cell data and rendering the worksheet.

![Screenshot 2022-12-20 at 10 47 14 AM](https://user-images.githubusercontent.com/6747531/208589408-c43260c1-e878-40c1-995c-3d334bc3eade.png)

**Comment by shuanhukexuejia** [comment]
Can you provide some specific values (the row[column]), for us to debug?

**Comment by sandilya28** [comment]
Here's the CSV file for the output rendering on the worksheet

[rowsncolumns.csv](https://github.com/ruilisi/fortune-sheet/files/10265832/rowsncolumns.csv)

**Comment by shuanhukexuejia** [comment]
Maybe you can just show us the log of celldata, it's more easier for us to debug

**Comment by sandilya28** [comment]
```
[
  {
    "r": 0,
    "c": 0,
    "v": "main"
  },
  {
    "r": 0,
    "c": 1,
    "v": "python_map_function"
  },
  {
    "r": 0,
    "c": 2,
    "v": "table"
  },
  {
    "r": 0,
    "c": 3,
    "v": "None"
  },
  {
    "r": 0,
    "c": 4,
    "v": "None"
  },
  {
    "r": 0,
    "c": 5,
    "v": "[col0, col1]"
  },
  {
    "r": 0,
    "c": 6,
    "v": "[TABLE, POINTER]"
  },
  {
    "r": 0,
    "c": 7,
    "v": "None"
  },
  {
    "r": 0,
    "c": 8,
    "v": "None"
  },
  {
    "r": 0,
    "c": 9,
    "v": "None"
  },
  {
    "r": 0,
    "c": 10,
    "v": "1286"
  },
  {
    "r": 1,
    "c": 0,
    "v": "main"
  },
  {
    "r": 1,
    "c": 1,
    "v": "seq_scan"
  },
  {
    "r": 1,
    "c": 2,
    "v": "table"
  },
  {
    "r": 1,
    "c": 3,
    "v": "None"
  },
  {
    "r": 1,
    "c": 4,
    "v": "None"
  },
  {
    "r": 1,
    "c": 5,
    "v": "[]"
  },
  {
    "r": 1,
    "c": 6,
    "v": "[]"
  },
  {
    "r": 1,
    "c": 7,
    "v": "None"
  },
  {
    "r": 1,
    "c": 8,
    "v": "None"
  },
  {
    "r": 1,
    "c": 9,
    "v": "None"
  },
  {
    "r": 1,
    "c": 10,
    "v": "3"
  },
  {
    "r": 2,
    "c": 0,
    "v": "main"
  },
  {
    "r": 2,
    "c": 1,
    "v": "index_scan"
  },
  {
    "r": 2,
    "c": 2,
    "v": "table"
  },
  {
    "r": 2,
    "c": 3,
    "v": "None"
  },
  {
    "r": 2,
    "c": 4,
    "v": "None"
  },
  {
    "r": 2,
    "c": 5,
    "v": "[]"
  },
  {
    "r": 2,
    "c": 6,
    "v": "[]"
  },
  {
    "r": 2,
    "c": 7,
    "v": "None"
  },
  {
    "r": 2,
    "c": 8,
    "v": "None"
  },
  {
    "r": 2,
    "c": 9,
    "v": "None"
  },
  {
    "r": 2,
    "c": 10,
    "v": "5"
  },
  {
    "r": 3,
    "c": 0,
    "v": "main"
  },
  {
    "r": 3,
    "c": 1,
    "v": "pragma_version"
  },
  {
    "r": 3,
    "c": 2,
    "v": "table"
  },
  {
    "r": 3,
    "c": 3,
    "v": "None"
  },
  {
    "r": 3,
    "c": 4,
    "v": "None"
  },
  {
    "r": 3,
    "c": 5,
    "v": "[]"
  },
  {
    "r": 3,
    "c": 6,
    "v": "[]"
  },
  {
    "r": 3,
    "c": 7,
    "v": "None"
  },
  {
    "r": 3,
    "c": 8,
    "v": "None"
  },
  {
    "r": 3,
    "c": 9,
    "v": "None"
  },
  {
    "r": 3,
    "c": 10,
    "v": "7"
  },
  {
    "r": 4,
    "c": 0,
    "v": "main"
  },
  {
    "r": 4,
    "c": 1,
    "v": "pragma_functions"
  },
  {
    "r": 4,
    "c": 2,
    "v": "table"
  },
  {
    "r": 4,
    "c": 3,
    "v": "None"
  },
  {
    "r": 4,
    "c": 4,
    "v": "None"
  },
  {
    "r": 4,
    "c": 5,
    "v": "[]"
  },
  {
    "r": 4,
    "c": 6,
    "v": "[]"
  },
  {
    "r": 4,
    "c": 7,
    "v": "None"
  },
  {
    "r": 4,
    "c": 8,
    "v": "None"
  },
  {
    "r": 4,
    "c": 9,
    "v": "None"
  },
  {
    "r": 4,
    "c": 10,
    "v": "9"
  }
]
```

**Comment by shuanhukexuejia** [comment]
Have found the reason: 
doesn't check the type of value
<img width="212" alt="image" src="https://user-images.githubusercontent.com/84694078/208606370-a70a4d1f-d320-456b-8065-7903d7f18889.png">

you can fix it temporarily by adding default foramt
```
celldata.map((cell) => ({
    ...cell,
    v: {
      ct: {
        fa: "General",
        t: "n",
      },
      v: cell.v,
      m: cell.v.toString(),
    },
  }))
```

**Comment by sandilya28** [comment]
Awesome!! That works! @shuanhukexuejia Thank you.

**Comment by shuanhukexuejia** [comment]
* Has been fixed in #141
